src/typed-method-types/workflows/triggers/event-data/dnd_updated.ts

```diff
@@ -0,0 +1,33 @@
+const DNDStatus = {
```

Comment:
nit: Calling out that this is PascalCase while in other instances we are using snake_case (`shared_channel_invite_*` events, `user_joined_team` event)

PascalCase gives us consistency with the casing we use for the events themselves while snake_case gives us convenience when referencing via object property shorthand. I'd like to do one or the other and am slightly leaning towards PascalCase for consistency purposes